packages/stark-ui/src/modules/date-picker/components/date-picker.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-form-field>
+	<input matInput 
+		[disabled]="isDisabled"
+		[matDatepicker]="picker" 
```

Comment:
What happens if we want to use multiple `stark-date-picker` components on the same page ?
I think that the id of the date pickers will be the same and that means no date-picker at all will work. 

As it was the case in the old Stark, I think we should provide a binding for setting the name/id of the date-picker to allow to use multiple ones in the same HTML page

Reply:
Well true we had an id before. 
The reason why is that we won't be able to use any functions on the mat-date-picker getting it by id. Instead of that, the StarkDatePicker expose a public picker: MatDatepicker and a public pickerInput: MatDatepickerInput, so the end user have access to them in the TS and use the API provided by Material.
https://material.angular.io/components/datepicker/api

Reply:
Yep but the issue is not about interacting with the date-picker programatically, it is more about 2 other aspects as far as I can remember:
- the default HTML behaviour. In case the datepicker is inside a `<form>`, whenever such form is submitted, the fields of the form will be sent with the name of the `name` attribute of the `<input>` element. I think the same will be important in case we link an Angular Form Control with the datepicker.
- for the e2e testing, it would be easier to select a datepicker based on its `id` or `name` rather than with the CSS selector which might change more often than the attributes.

Reply:
The input name is already set:
```
@Input()
public pickerName: string = "";
```

About the id, true that it will be useful for e2e :-)